.github/workflows/migrator-comment.yml

```
@@ -40,11 +40,12 @@ jobs:
             exit 0
           fi
 
-          filesWithComment=$(git diff --unified=0 --diff-filter=d "$commits" -- $(echo $files) | grep "^\+.*$COMMENT")
+          filesWithComment=$(git diff --unified=0 --diff-filter=d "$commits" -- $(echo $files) | grep "^\+.*$COMMENT" || true)
```

```suggestion
          filesWithComment=$(git diff --unified=0 --diff-filter=d "$commits" -- $(echo $files) | grep -s "^\+.*$COMMENT")
```

Thoughts on using the `-s` option here?

> -s, --no-messages
>   Silent mode.  Nonexistent and unreadable files are ignored (i.e.,
>   their error messages are suppressed).